packages/react-components/src/components/Drawer/DrawerMenuItem.tsx

```diff
@@ -49,6 +50,10 @@ export interface _DrawerMenuItemProps {
    * A unique id set for the standard HTML id attibute
    */
   id?: string;
+  /**
+   * Property to set the focus since this is the first item in the menu
+   */
+  shouldFocusFirst?: boolean;
```

Comment:
nit: `shouldFocusOnMount` instead? As this is a single menuItem, it doesn't really have the concept of "first"